modules/renderer/Renderer.js

```diff
@@ -114,15 +114,15 @@ export default function Renderer(config = { }) {
      * @param {FontFace} fontFace - fontFace which gets rendered
      * @return {string} fontFamily reference
      */
-    renderFont(family, files, properties = { }) {
+    renderFont(family, files, props = { }) {
```

Comment:
Ah ok. I can revert that, not a problem - are you cool with the rest of the changes?